js/file/shared/File.svelte

```diff
@@ -20,7 +20,7 @@
 	label={label || "File"}
 />
 
-{#if value && (Array.isArray(value) ? value.length > 0 : true)}
+{#if (Array.isArray(value) && value.length > 0) || value !== null}
```

Comment:
#7132 has already been fixed so I don't think we need to make this change?

Reply:
Either is fine, but I can remove my change.

Reply:
The original seems slightly simpler to parse and we've already validated it, so maybe keep the original? Not a big deal either way